src/main/java/com/ldtteam/structurize/client/BlueprintRenderer.java

```diff
@@ -194,6 +195,7 @@ private void init(final BlueprintPreviewData previewData, final Map<Object, Exce
         }
 
         blockAccess.setSolidSubstitutionOverride(null);
+        blockAccess.setRenderBlocksNiceOverride(Structurize.getConfig().getClient() != null && Structurize.getConfig().getClient().renderPlaceholdersNice.get());
```

Comment:
This is client only code, same below

```suggestion
        blockAccess.setRenderBlocksNiceOverride(Structurize.getConfig().getClient().renderPlaceholdersNice.get());
```

Reply:
It's still in the other file, mergeable